build_deploy.sh

```diff
@@ -0,0 +1,33 @@
+set -exv
```

Comment:
```suggestion
#!/bin/bash
set -e
```
I would add shebang even if it does not have executable permissions.

Update `set -exv` by `set -e` before merge to avoid traces about the passwords are printed out in logs. Unless I am missing something about the execution of this script.

Reply:
executable permissions are a good point - I'm going to modify perms to make it executable, not sure if the Jenkins jobs are setup to do that automatically. Will just mirror what other service repos set for permissions.

The script is intended to be used by ci-int/ci-ext jenkins instances, ack on updating the set args.